### ISSUE TITLE
MIA: writing output files in "derived_data" folder

### ISSUE THREAD

**Issue by M40V** [M40V/populse_mia#25]
<a href="https://github.com/servoz"><img src="https://avatars1.githubusercontent.com/u/37933899?v=4" align="left" width="96" height="96" hspace="10"></img></a> **Issue by [servoz](https://github.com/servoz)**
_Wednesday Oct 17, 2018 at 11:45 GMT_
_Originally opened as https://github.com/populse/populse_mia/issues/25_

----

Some posts exist on this populse_sandbox issue (populse/populse_sandbox#5).
To sum up:
- It is not easy for all the platforms ... :smile: 

<br/><br/><br/><br/><br/><br/>

---
This is a copy of populse/populse_sandbox#5. Do not forget to close concomitantly the populse/populse_sandbox#5 and this issue !_

**Comment by M40V** [comment]
<a href="https://github.com/dhrbn"><img src="https://avatars1.githubusercontent.com/u/37616080?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [dhrbn](https://github.com/dhrbn)**
_Friday Oct 26, 2018 at 15:04 GMT_

----

Nipype already copies output files from its working directory to the right directory, maybe we could examine this in greater depth
